bot-double-inner-ring/src/test/java/osm/bots/innerringbot/utils/UserAuthorizationTest.java

```diff
@@ -0,0 +1,84 @@
+package osm.bots.innerringbot.utils;
+
+import de.westnordost.osmapi.OsmConnection;
+import de.westnordost.osmapi.map.MapDataApi;
+import de.westnordost.osmapi.map.data.Relation;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import oauth.signpost.OAuthConsumer;
+import oauth.signpost.basic.DefaultOAuthConsumer;
+import oauth.signpost.basic.DefaultOAuthProvider;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Slf4j
+@Disabled
+class UserAuthorizationTest {
+
+    private static final String CONSUMER_KEY = "consumer_key";
+    private static final String CONSUMER_SECRET_KEY = "consumer_secret_key";
+    private static final String CONSUMER_TOKEN = "consumer_token";
+    private static final String CONSUMER_TOKEN_SECRET = "consumer_token_secret";
+
+    private static final String REQUEST_TOKEN_URL = "https://www.openstreetmap.org/oauth/request_token";
+    private static final String ACCESS_TOKEN_URL = "https://www.openstreetmap.org/oauth/access_token";
+    private static final String AUTHORIZATION_WEBSITE_URL = "https://www.openstreetmap.org/oauth/authorize";
+    private static final String OPENSTREETMAP_ORG_API_0_6 = "https://api.openstreetmap.org/api/0.6/";
+
+    private static final long EXAMPLE_RELATION_ID = 10633691L;
+
+    /**
+     * This test for provided consumer-key and consumer-secret-key, allows to retrieve consumer-token and consumer-secret,
+     * which later on can be used for authorized requests to OSM.
+     * Required two parameters:
```

Comment:
I feel like specifying count of parameters is redundant here 
```suggestion
     * Required parameters:
```